tests/periph_pwm/main.c

```diff
@@ -68,12 +167,94 @@ int main(void)
         }
 
         state += step;
-        if (state <= 0 || state >= (int)STEPS) {
+        if (state <= 0 || state >= (int)OSC_STEPS) {
             step = -step;
         }
 
-        xtimer_periodic_wakeup(&last_wakeup, INTERVAL);
+        xtimer_periodic_wakeup(&last_wakeup, OSC_INTERVAL);
     }
 
     return 0;
 }
+
+int power(int argc, char** argv){
+    if (argc != 3) {
+        printf("usage: %s <dev> <state>\n", argv[0]);
+        printf("\tdev: device by number between 0 and %d\n", PWM_NUMOF-1);
+        printf("\tstate:\n");
```

Comment:
you could/should use `puts()` for printing strings without formatting parameters (slightly more efficient...).